scripts/services/getConfig.ts

```diff
@@ -0,0 +1,145 @@
+                  "tag": {
+                    "id": "type",
+                    "name": "Type",
+                    "description": "- Internal: The Lightning Network Daemon service installed to your Embassy.\n- External: A Lightning Network Daemon instance running on a remote device (advanced).\n",
+                    "variant-names": {
+                        "internal": "Internal",
+                        "external": "External"
+                    }
+                  },
+                  "default": "internal",
+                  "variants": {
+                    "internal": {},
+                    "external": {
+                        "address": {
+                          "type": "string" as const,
+                          "name": "Public Address",
+                          "description": "The public address of your LND REST server\nNOTE: RTL does not support a .onion URL here\n",
+                          "nullable": false
+                        },
+                        "rest-port": {
+                          "type": "number" as const,
+                          "name": "REST Port",
+                          "description": "The port that your Lightning Network Daemon REST server is bound to",
+                          "nullable": false,
+                          "range": "[0,65535]",
+                          "integral": true,
+                          "default": 8080
+                        },
+                        "macaroon": {
+                          "type": "string" as const,
+                          "name": "Macaroon",
+                          "description": "Your admin.macaroon file, Base64URL encoded. This is the same as the value after \"macaroon=\" in your lndconnect URL.",
+                          "nullable": false,
+                          "masked": true
+                        }
+                    }
+                  }
+              }
+            },
+            "c-lightning": {
+              "name": {
+                  "type": "string" as const,
+                  "name": "Node Name",
+                  "description": "Name of this node in the list",
+                  "default": "Embassy CLN",
+                  "nullable": false
+              },
+              "connection-settings": {
+                  "type": "union" as const,
+                  "name": "Connection Settings",
+                  "description": "The Core Lightning (CLN) node to connect to.",
+                  "tag": {
+                    "id": "type",
+                    "name": "Type",
+                    "description": "- Internal: The Core Lightning (CLN) service installed to your Embassy.\n- External: A Core Lightning (CLN) instance running on a remote device (advanced).\n",
+                    "variant-names": {
+                        "internal": "Internal",
+                        "external": "External"
+                    }
```

Comment:
Thought we were deprecating external nodes

Reply:
see above